content/doc/developer/publishing/plugin-site.adoc

```diff
@@ -1,8 +1,109 @@
 ---
 title: Plugin Site
 layout: developersection
-wip: true
-references:
-- url: https://wiki.jenkins.io/display/JENKINS/Plugin+Site+Information+for+Plugin+Developers
-  title: Plugin site information on the wiki
 ---
+
+This page explains how https://plugins.jenkins.io/[the plugins site]
+works. The intended audience is plugin developers.
+
+This is not a manual on how to _use_ the site to e.g. find plugins, but
+rather explains how it gets its data and how plugin developers can
+change the data on it.
+
+== Data shown on the plugins site
+
+=== Which plugins are listed? How can I make my plugin appear?
+
+All plugins published to the current (weekly) update center are listed.
+
+=== What is the plugin installation trend based on?
+
+This uses the monthly data from http://stats.jenkins.io/ as gathered
+from Jenkins instances that have not opted out of reporting data.
+
+Learn more: link:../usage-statistics[usage statistics]
+
+=== What does "most installed" mean?
+
+This ranks plugins by the number of different Jenkins instances that
+reported the plugin installed during the previous month.
+
+=== What does 'Recently updated' mean?
+
+This is based on metadata created by the update site generator, which
+uses the creation date of a plugin release.
+
+Note that very recent plugin releases may not appear here due to caching
+and delays in data aggregation.
+
+=== What does 'Trending' mean?
+
+This is an attempt to capture the plugins whose popularity is growing
+rapidly. As we only have the monthly installation stats for this, it's
+necessarily a worse approximation than what you'd likely see e.g. in app
+stores.
+
+Currently (2016-11) this just divides the previous month's install count
+by the month before that. Plugins with the greatest numbers are shown in
+descending order.
+
+=== What are categories based on?
+
+Categories are based on the labels on plugin wiki pages. Those with the
+`+plugin-+` prefix are passed through update site generation and will
+appear in update-center.json (which gets downloaded by Jenkins, where
+these labels also serve to group plugins in the Plugin Manager).
+
+The categories are filtered, grouped, and given nicer names based on
+plugin site configuration here:
+https://github.com/jenkins-infra/plugin-site-api/blob/master/src/main/resources/categories.json
+
+Some categories that we expect wouldn't mean a lot to users who aren't
+plugin developers themselves have been filtered out ("plugin
+wrappers"?).
+
+=== Where does the description of the plugin come from?
+
+The plugin site will pull from either the plugin's README page on GitHub or it's wiki page.
+
+==== GitHub
+See link:../documentation/#using-github-as-a-source-of-documentation[documentation] for 
+how to configure GitHub documentation.
```

Comment:
As a general recommendation, it would be nicer to review PRs like this one if you separate the raw import (+ potential formatting adjustments as needed) and content corrections in the PR into different commits.